x-pack/plugins/apm/server/routes/service_groups/get_service_group_alerts.ts

```diff
@@ -66,17 +63,7 @@ export async function getServiceGroupAlerts({
   };
   const result = await apmAlertsClient.search(params);
 
-  interface ServiceGroupsAggResponse {
-    buckets: Record<
-      string,
-      AggregationsFilterAggregate & {
-        alerts_count: AggregationsCardinalityAggregate;
-      }
-    >;
-  }
-
-  const { buckets: filterAggBuckets } = (result.aggregations
-    ?.service_groups ?? { buckets: {} }) as ServiceGroupsAggResponse;
+  const filterAggBuckets = result.aggregations?.service_groups.buckets ?? {};
```

Comment:
Are `service_groups` always defined? By the code above it seemed they weren't always defined

Reply:
`service_groups` is the named aggregation. We are optional chaining it both in the old and new one